### DISCUSSION TITLE
[feature request] Webp support

### DISCUSSION THREAD

**Discussion by HamadTheIronside** [django-cms/django-filer#1298] [General]
Hey team Django-filer,
Is it possible to add webp support to the Image model? as 95% of the browsers support it, and it's optimized and better than the others.

I would be happy to do a PR to make it happen.

By the way, thanks for the library ❤️

**Comment by jrief** [comment]
There is already a workaround on how to handle WEBP in django-filer. We currently do not fully rely on WEBP, because Safari (= the new Internet Explorer) still doesn't support it.

The point is, that most users upload their images in JPEG and **django-filer** currently always converts into the same image format as its source. Therefore, WEBP can be an additional alternative but not a replacement. And for this, please use the mentioned workaround.
